### ISSUE TITLE
Create Ensure-PSModule Operation

### ISSUE THREAD

**Issue by rhessinger** [Inedo/inedox-windows#86]
The `Ensure-PSModule` operation is to ensure that a Powershell module is installed on your machine.  It is basically a wrapper to the `Install-Module` powershell command but adds a check to verify the version is installed. 

```
Ensure-PSModule {
   ModuleName: ...
   Version: ...
   MinVersion:...
   MaxVersion:...
   Force: false
   AllowClobber: false
   Repository: ... 
   Credential: ...(I expect this to be a ResourceCredential)
   Scope:...
   Properties: ... (any other additional properties to pass to Install-Module)
}
```

Where it just ensures that version (or within the min/max) is installed. 

You would specify Version or Min/Max Version but not both.

**Comment by rhessinger** [comment]
For more information see forums thread: https://forums.inedo.com/topic/3036/ensure-dsc-and-module-version

**Comment by rhessinger** [comment]
The PowerShell operations have been migrated to [inedox-scripting](https://github.com/Inedo/inedox-scripting) and this operation was included in that starting in version 1.10.  Closing this issue since it is no longer in this extension,
